### ISSUE TITLE
[EmoteKeys][Bug] MissingMethodException

### ISSUE THREAD

**Issue by ion-tichy** [aedenthorn/ValheimMods#60]
Valheim Version: 0.203.11
BepInEx: 5.4.16.0
EmoteKeys: 0.2.0
OS: Win7

During startup, I get the following error message in console:

```
MissingMethodException: void Player.StartEmote(string,bool)
```

The default Numpad mapping does not work either.

**Comment by ion-tichy** [comment]
Reviewed the code and it seems that the dll on Nexus Mods is simply outdated?

**Comment by aedenthorn** [comment]
I don't think this is true, the code hasn't actually changed, and that method does exist. I have no problem using the mod.
